src/include/uapi/ipc.h

```diff
@@ -243,6 +244,20 @@ struct sof_ipc_compound_hdr {
 #define SOF_DAI_INTEL_SSP_FRAME_PULSE_WIDTH_MAX		38
 #define SOF_DAI_INTEL_SSP_SLOT_PADDING_MAX		31
 
+/* SSP clocks control settings
+ *
+ * Macros for clks_control field in sof_ipc_dai_ssp_params struct.
+ */
+
+/* mclk keep active */
```

Comment:
I would like to add an MCLK_0/1_ENABLE to be able to control the SSP mdivc (or mdivr?) register

Reply:
Do you mean controlling mdivc register insted of:
mdivc |= 0x1;?
I could add this in another patch.

Reply:
yes that's what I meant. You don't need to enable the outputs on all devices, some don't even have a physical pin.